src/web/mjs/engine/Storage.mjs

```diff
@@ -869,4 +878,40 @@ export default class Storage {
         }
         return undefined;
     }
+
+    /**
+     * HYPOFLEX
+     * Save the given value for the given key in the bookmark storage
+     */
+    saveBookmarks( key, value, indentation ) {
+        return new Promise( ( resolve, reject ) => {
+            this.fs.writeFile( Engine.Settings.bookmarkDirectory.value + '\\hakuneko.' + key, JSON.stringify( value, undefined, indentation ), function( error ) {
```

Comment:
I think storage provides `path` or `this.path` so your separator issue can now be solved with `path.join(Engine.Settings.bookmarkDirectory.value, 'hakuneko.' + key)`

Reply:
`\\hakuneko` is indeed windows only, since windows needs a backslash and linux/osx needs a forward slash.

i was working on a switch case based on variable

```javascript
let platform = undefined;
switch(this.platform.indexOf){
case 'win':
 platform = '\\hakuneko'
 break;
case 'darwin':
 platform = '/hakuneko'
break;
}
```